engine/dlib/src/dlib/log.cpp

```diff
@@ -469,7 +502,7 @@ void LogInternal(Severity severity, const char* domain, const char* format, ...)
         return;
     }
 
-    if (severity < g_LogLevel)
+    if (severity < dmLog::g_LogLevel)
```

Comment:
I'm not happy of the mix of namespace/non-namespace inside log.cpp, but it will have to do for now.